interpreter.js

```diff
@@ -146,6 +146,20 @@ Interpreter.READONLY_NONENUMERABLE_DESCRIPTOR = {
 // Since this is for atomic actions only, it can be a class property.
 Interpreter.toStringCycles_ = [];
 
+/**
+ * Creates an abstract syntax tree from a string of code
+ * @param {string} code - Raw JavaScript text
+ * @param {object} parseOptions - Configuration for acorn parser
+ * @returns {object} the AST for the given code
+ */
+Interpreter.generateAST = function(code, parseOptions) {
+  var options = Interpreter.PARSE_OPTIONS;
+  Object.keys(parseOptions).forEach(function(key) {
```

Comment:
Could this be written as just `var options = {...Interpreter.PARSE_OPTIONs, ...parseOptions}`?

Reply:
Good call, but unfortunately it can't. This repo keeps ES5 standards rather than ES6 standards.